app/presenters/export_summary.rb

```diff
@@ -14,6 +14,11 @@ class ExportSummary
     prefix: true
   )
 
+  delegate(
+    :account_notes,
+    to: :account
+  )
```

Comment:
I did this without the prefix, because otherwise it's be `account_account_notes` which felt wonky compared to the other methods.